third-party/llvm/Makefile.include-llvm

```diff
@@ -2,11 +2,17 @@ include $(THIRD_PARTY_DIR)/llvm/Makefile.share-included
 
 # Enable RV and Polly if they are in the source directory
 LLVM_RV_FILE = $(LLVM_DIR)/$(LLVM_SUBDIR)/tools/rv/
```

Comment:
Are we keeping this for backwards compatibility?

Reply:
I left it in because I'm not sure if it will be needed or not in the future if we start to bundle RV.